docs/Examples/OptionalParameters.md

```diff
@@ -42,3 +42,24 @@ extension MyService: TargetType {
 //...
 }
 ```
+
+
+Important Note
+--------------
+You **have to** add optional parameters like shown above, one per line. Optional parameters won't be removed in case of ```nil``` if you try to initialize them within one line, e.g.:
+
+```swift
+//...
+	public var parameters: [String: Any]? {
+	    switch self {
+	    case .users(let limit):
+	        let params: [String: Any] = ["limit": limit]
```

Comment:
Since this code is a demonstration of how _not_ to do things, adding a comment like `// This won't work!` above this line might be a good idea. Thoughts?